demo/admin/src/products/future/generated/ProductForm.tsx

```diff
@@ -283,7 +299,9 @@ export function ProductForm({ id }: FormProps): React.ReactElement {
                                     });
                                     return data.productCategories.nodes;
                                 }}
-                                getOptionLabel={(option) => option.title}
+                                // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+                                // @ts-ignore
+                                getOptionLabel={(option) => option.title} // option is of type unknown
```

Comment:
Regenerate the generated types, then this works perfectly fine. (If you'd used `@ts-expect-error`, then it would be have become visible in the lint workflow).